handbook/company/communications.md

```diff
@@ -154,16 +143,34 @@ Team members can request reimbursement through [Gusto]([https://app.gusto.com/ex
+### Brex
+All Fleeties will be provided with a Brex account and virtual Brex card when they onboard at Fleet. Fleeties will be prompted to order a physical Brex card when they log in to their account. Fleet recommends you order a physical card as soon as possible.
 
+#### Individualized expenses
+Recurring expenses related to a particular team member, such as coworking fees, are called _individualized expenses_.  These expenses are also considered [non-personnel expenses](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278), with a few extra considerations:
 - Non-recurring (one-off) expenses such as an Uber ride from the airport are NOT considered "individualized".
 - Seat licenses for tools like Salesforce or GitHub are NOT considered "individualized".
 - Individualized expenses should include the team member's name explicitly in the name of the expense.
 - If multiple team members use the same vendor for an individualized expense (for example, "WeWork"), use a separate row for each individualized expense.  (For example: "Coworking, Mungojerry (WeWork)" and "Coworking, Jennyanydots (WeWork)")
 - Individualized expenses are always attributed to the  "🔦 Business operations" department.
-- These expenses are still considered non-personnel expenses, in the same way seat licenses for tools like Salesforce or GitHub are considered non-personnel expenses.   
+- These expenses are still considered non-personnel expenses, in the same way seat licenses for tools like Salesforce or GitHub are considered non-personnel expenses.
 
+#### Travel expenses
+Travel expenses are any purchases made that support the attendance of a Fleetie at a company event, customer visit, conference attendance, or other work-required travel. These expenses are considered [non-personnel expenses](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278), and are not "individualized". Travel expenses are allocated to a specific event spend. In order to allocate this spend correctly, Fleeties must follow the [expense acquittal process](#expense-aquittal).
+For more guidance on travel, see [Company travel](#company-travel).
+
+#### One-off expenses
+One-off expenses are non-travel, non-recurring expenses. These expenses are considered [non-personnel expenses](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278). There are a varity of reasons a Fleetie might make a one-off expense, including purchase of headphones or ergonomic workstation needs, printing or sending of work documents, purchase of a swag item, etc. 
+One-off expenses should always be made with the Fleetie's Brex card, and aquitted using the [expense acquittal process](#expense-aquittal)
```

Comment:
Either a newline between paragraphs, or put this on the same line as the previous

Reply:
`#expense-aquittal` would be a broken link